### DISCUSSION TITLE
do notes titles matter a lot  ?

### DISCUSSION THREAD

**Discussion by avichou** [brianpetro/obsidian-smart-connections#570] [Q&A] [answered]
long story short im trying to decide whether to setup automatic backup from upnote , or to use the export function

the main difference between the two lies in the filenames: auto backup is UUID , and export (manual export) has the proper filenames and titles 

here's an example of UUID titles ; 0a2ced31-1230-4b50-882e-9bebe04c73c5 

my question is, is the title important when it comes to searching for answers and asking questions to your vault ? or is it not rly that bad so long as the actual note and all the content is properly rendered. basically only the title looks bad 

<img width="787" alt="image" src="https://github.com/brianpetro/obsidian-smart-connections/assets/86660671/711d3f29-7c37-4140-8361-4f5490ae4a9b">

**Comment by brianpetro** [comment, marked as answer]
Hi @avichou 

If all notes in your vault have that title format, then it may not impact much. 

However, if only some notes have the heading, then notes that share the same format of heading will be more likely to be surfaced as relevant. 

🌴

**Reply by avichou** [reply]
> Hi @avichou 
> 
> If all notes in your vault have that title format, then it may not impact much. 
> 
> However, if only some notes have the heading, then notes that share the same format of heading will be more likely to be surfaced as relevant. 
> 
> 🌴

Oh wow I hadn't thought of that . Gosh darn . 


Guess I'll have to decide between export or backup

Speaking of if you have any idea idea on how to make a database or sth out of my upnote notes pls lmk ;) 

Thank you so much for your beautiful work man btw , may you be blessed !

**Reply by brianpetro** [reply]
Unfortunately, I'm not familiar with Upnote. But maybe someone else from the community will see this and give you some tips 😊 

And it's always a good idea to search the Obsidian community plugins directory for other plugins that might exist to help.

🌴

**Reply by avichou** [reply]
thank you . I apologize, my question was out of place . thanks for being so nice and giving a kind answer ;) 

I must say, I have other questions directly related to your plugin/the under the hood mechanisms, Im writing them but you don't owe me an answer or anything ! 

1), when performing a search through all of your notes , does it base its research starting from the very note you're in? if so , it would indeed make a difference, even if it's global search, as it is searching from this note as a basis. 
basically, since you need to start copilot from a note , lets say a note called "neurosis part 2" , will the results and search depend on it, or is it truly global and unrelated from the AI chat trigger  ? if such, UUID might not be good 



2) next on embedding,I have all my 800 notes in md in obsidian. however I use another note taking app , from which I export from time to time and import into obsidian, to use your plugin. 

then a month later, i do another export of my quick note taking app (upnote), which now contains 900 notes . 800+100 news , plus sporadic changes brought to a few notes (let's say 50)

if i drag and drop all the notes in finder/file management app, it'll merge them i think?. now, will embedding be required to run on 900 (nay 950?) , or will it only update and embed the new notes , aka 100 only , whilst keeping the other 800 embedding (thus saving on cost and energy) ? 

it's a big question sorryyyy
